### PR TITLE
Style searchbox and table

### PR DESCRIPTION
**Pull request by jialin-wu-02**
1. Using css flex box to style the search box.
2. Update the style of the table.
3. Move the "# of source displaying" into the `SourceList` component.

### CODE REVIEW

**Comment by stefanv** [comment]
Thanks, Alex.  The errors are related: you can check your code formatting by doing `make lint`.

**Comment by acrellin** [comment]
@jialin-wu-02  Thanks!  Please go ahead and just modify the original `SearchBox` component (and css file, if relevant) instead of adding a new `NewSearchBox` component.

**Comment by jialin-wu-02** [comment]
Just passed all checks, and there is no error in the local server either.

**Comment by stefanv** [comment]
Thanks, Alex, this is looking great!

I see that, currently, you have one Custom component that itself switches out to, e.g., a time formatted box if format == UTC.  We have to write the JSX anyway, so how about making each of those separate components: TimeFormatInput, etc. and then explicitly invoking them in the JSX?

**Comment by jialin-wu-02** [comment]
Fixed.

File: static/js/components/SearchBox.jsx [hunk 1]
```diff
@@ -4,6 +4,13 @@ import { useDispatch } from 'react-redux';
 
 import * as Actions from '../ducks/sources';
 
+import styles from "./SearchBox.css";
+
+// Custom Components:
```
Comment (acrellin) [inline static/js/components/SearchBox.jsx]: Unnecessary comment

File: static/js/components/CheckBox.jsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styles from './CheckBox.css';
+
+
+function CheckBox({ label, name, checked, onChange, size }) {
```
Comment (acrellin) [inline static/js/components/CheckBox.jsx]: For consistency, this could be an arrow function

**Comment by jialin-wu-02** [comment]
- [ ] Thumbnail jump to another row while scaling.

**Comment by stefanv** [comment]
Will be re-done using Material UI now

**Comment by acrellin** [comment]
Closed in favor of https://github.com/skyportal/skyportal/pull/375

**Pull request closed without merging**